packages/ironic/charts/templates/certificates.yaml

```diff
@@ -38,6 +38,7 @@ spec:
     kind: Issuer
     name: ca-issuer
   secretName: ironic-inspector-cert
+{{- if .Values.global.enable_vmedia_tls -}}
```

Comment:
I think the whitespace control here is wrong - in my environment it ends up like `secretName: ironic-inspector-cert---` - I think removing one of the `-` characters in this expression will fix it

Reply:
Fixed, thanks!